src/comment/models.py

```diff
@@ -31,6 +31,5 @@ class Comment(Base):
     )
     author = relationship("Author", back_populates="comment")
 
-    article_id = Column(
-        Integer, ForeignKey("article.id"), name="article_comment"
-    )
+    article_id = Column(Integer, ForeignKey("article.id"), unique=True)
+    article = relationship("Article", uselist=False, backref="comment")
```

Comment:
why uselist=False ?)))))))

Read the docs about it. Seems like chat is proposing you with this setting and it is not correct here